src/main/java/com/klodnicki/bike/rest/API/Bike/RentABikeRestApiApplication.java

```diff
@@ -3,6 +3,15 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+//Napisz aplikację w Spring Boot, która będzie pozwalać na wypożyczanie rowerów miejskich za pomocą mobilnej aplikacji
+// na smartphone (Coś jak hulajnogi elektryczne bolta, na przykład).
+//		W mieście mamy stacje rowerowe, które służą do odstawiania tam rowerów i wypożyczania ich stamtąd.
+//		Każda stacja ma przyjazną dla użytkownika nazwę, którą łatwo zidentyfikować i rozpoznać oraz ograniczoną ilość
+//		miejsc dla rowerów. Nie można zostawić przy takiej stacji roweru, jeśli wszystkie miejsca są zajęte.
+//		Każdy rower ma również swoją nazwę lub numer, który go identyfikuje. Np. H0123 - tak, aby użytkownik również
+//		miał łatwość rozpoznania roweru, który chce wypożyczyć z aplikacji.
+//		Rowery obecnie dostępne to rowery elektryczne i tradycyjne.
+
```

Comment:
I think trello is much better place to put this description. Or you can create a readme file for the project :) Unless you really need to have this in the code, I say it could change place to more suitable one :)

Reply:
haha okay, I have it in trello as well ;)
OK ready to delete it! ;)